### DISCUSSION TITLE
Is there a list of the accepted colour values for the git icons?

### DISCUSSION THREAD

**Discussion by hankertrix** [ibhagwan/fzf-lua#1645] [Q&A] [answered]
I'm trying to change the colours of my git icons, but I have no idea what is being accepted as a valid colour.

Invalid colours result in the window not displaying anything at all, so it would be great to know what is accepted and what isn't.

From my experimentation, hex colour codes are not supported, and a few colour names are supported, but I have no idea which ones are supported other than `green`, `yellow`, `red`, `blue`, `grey` and `magenta`.

It seems like `gray` must be spelt as `grey`.

**Comment by ibhagwan** [comment, marked as answer]
Since this is a terminal the accepted colors are the term ansi colors:
https://github.com/ibhagwan/fzf-lua/blob/e724f5080b94ae685978919c971b3371b1b48a60/lua/fzf-lua/utils.lua#L582-L599

Doesn’t mean you’re limited to these, you can use `utils.ansi_from_hl` to transform any text using a neovim highlight.

**Reply by hankertrix** [reply]
This is just what I need, thank you!
